src/theory/bv/slicer.cpp

```diff
@@ -381,12 +403,12 @@ void UnionFind::alignSlicings(const ExtractTerm& term1, const ExtractTerm& term2
 
   Assert (nf1.base.getBitwidth() == nf2.base.getBitwidth());
   
-  // first check if the two have any common slices 
+  // first check if the two have any common slices
   std::vector<TermId> intersection; 
-  utils::intersect(nf1.decomp, nf2.decomp, intersection); 
-  for (unsigned i = 0; i < intersection.size(); ++i) {
-    // handle common slice may change the normal form 
```

Comment:
Is there a reason why you removed the comment?

Reply:
Oh. No. Accident. Will readd.